config/initializers/secure_headers.rb

```diff
@@ -46,6 +46,10 @@
 
   default_csp_config[:script_src] = ["'self'", "'unsafe-eval'"] if !Rails.env.production?
 
+  if IdentityConfig.store.rails_mailer_previews_enabled
+    default_csp_config[:style_src] = ["'self'", "'unsafe-inline'"]
```

Comment:
Should we merge this to the existing value, which can include `IdentityConfig.store.asset_host` ?

```suggestion
    default_csp_config[:style_src] << "'unsafe-inline'"
```

Reply:
oh yes that would make more sense